apps/api-documenter/package.json

```diff
@@ -27,7 +27,7 @@
     "@types/colors": "1.1.3",
     "@types/fs-extra": "0.0.37",
     "@types/js-yaml": "3.9.1",
-    "@types/node": "6.0.88",
+    "@types/node": "^8.0.0",
```

Comment:
>"@types/node": "^8.0.0", [](start = 4, length = 24)

We have to use node 8 with PNPM?

Reply:
Nope. reverted

---
In reply to: [159794698](https://github.com/Microsoft/web-build-tools/pull/383#discussion_r159794698) [](ancestors = 159794698)